drivers/zooz/zooz-zen32-scene-ctlr.groovy

```diff
@@ -396,22 +470,29 @@ List<String> setLED(Number ledNumber, String colorName, brightness) {
          intLevel = 0
    }
    List<String> cmds = []
-   if ((brightness as Integer) <= 0) {
+   if (brightness != null && (brightness as Integer) == 0) {
       // Set LED to "always off" (may want to change in future if add association), unless #5/relay and configured not to:
-      if ((ledNumber as Integer) != 5 || ((ledNumber as Integer) == 5 && (relayLEDBehavior as Integer == 2))) {
-         cmds << zwaveSecureEncap(zwave.configurationV1.configurationSet(scaledConfigurationValue: 2, parameterNumber: ledIndicatorParams[ledNumber as Integer], size: 1))
-      }
-      else {
-         if (enableDebug) log.debug "LED number is 5 and level is 0 but configured to not allow turning off"
+      if (intLedNum != 5 || (relayLEDBehavior as Integer) == 4) {
```

Comment:
This if clause was just simplified from original and made to work with new setting options